api/controllers/users.js

```diff
@@ -1,24 +1,61 @@
 const User = require('../models/user');
+const cloudinary = require('cloudinary').v2;
+require('dotenv').config();
+
+cloudinary.config({
+  cloud_name: 'de2y4osyf',
+  api_key: process.env.API_KEY,
+  api_secret: process.env.API_SECRET,
+});
 
 const UsersController = {
   Create: (req, res) => {
-    const user = new User({
-      email: req.body.email,
-      password: req.body.password,
-      display_name: req.body.display_name,
-    });
-
+    const seed = Math.round(Math.random() * 100);
```

Comment:
To my understanding, this seed defines how the image is saved by assigning it a random integer between 0 and 100. Is it not possible, or even certain if we store > 101 images, that we will have duplicate seeds for certain images using this method?